internal/server/server.go

```diff
@@ -166,6 +166,10 @@ func (s *Server) handleCommand(client *Client, cmd string, arg string) {
 		client = NewClient(client.conn, client.ipAddr)
 		s.Clients = append(s.Clients, client)
 		s.BroadcastServerMessage(fmt.Sprintf("%s joined the chat", client.username))
+	case "/users":
+		users := len(s.Clients)
+		s.SendServerPrivateMessage(fmt.Sprintf("connected users :%d", users) , client)
+
```

Comment:
these are outdated but sorry for late push.

Reply:
To my surprise, you had already refactored the whole source code just after my PR, haha.